ops/ansible/roles/bfd-db-migrator/templates/migrator-monitor.sh.j2

```diff
@@ -65,18 +65,46 @@ function get_migrator_state {
 #######################################
 function migrator_is_running {
   local _state="$1"
-  local _code="$(jq -r .code <<< "$_state")"
-  local _status="$(jq -r .status <<< "$_state")"
+
+  local _code
+  local _status
+  _code="$(jq -r .code <<< "$_state")"
+  _status="$(jq -r .status <<< "$_state")"
   [ "$_code" = '(null)' ] && [ "$_status" = "0/0" ]
 }
 
+#######################################
+# Sets the migration schema version based on the monitor log
+# Globals:
+#   None
+# Arguments:
+#   Current state of the migrator
+# Outputs:
+#  json response to be used to update the schema version
+#  that was reported out in the monitor log and report that schema version
+#  to be used to send to SQS in order to update the migrator on whether to deploy or not
+#######################################
+function set_migrated_state {
```

Comment:
can we rename this to something like "get_migrator_schema_version"?

Reply:
done